keyboards/rocketboard_16/keycode_lookup.c

```diff
@@ -0,0 +1,426 @@
+ {"KC_LGUI", KC_LGUI},
+ {"KC_RCTL", KC_RCTL},
+ {"KC_RSFT", KC_RSFT},
+ {"KC_RALT", KC_RALT},
+ {"KC_RGUI", KC_RGUI},
+ {"KC_MS_UP", KC_MS_UP},
+ {"KC_MS_DOWN", KC_MS_DOWN},
+ {"KC_MS_LEFT", KC_MS_LEFT},
+ {"KC_MS_RIGHT", KC_MS_RIGHT},
+ {"KC_MS_BTN1", KC_MS_BTN1},
+ {"KC_MS_BTN2", KC_MS_BTN2},
+ {"KC_MS_BTN3", KC_MS_BTN3},
+ {"KC_MS_BTN4", KC_MS_BTN4},
+ {"KC_MS_BTN5", KC_MS_BTN5},
+ {"KC_MS_WH_UP", KC_MS_WH_UP},
+ {"KC_MS_WH_DOWN", KC_MS_WH_DOWN},
+ {"KC_MS_WH_LEFT", KC_MS_WH_LEFT},
+ {"KC_MS_WH_RIGHT", KC_MS_WH_RIGHT},
+ {"KC_MS_ACCEL0", KC_MS_ACCEL0},
+ {"KC_MS_ACCEL1", KC_MS_ACCEL1},
+ {"KC_MS_ACCEL2", KC_MS_ACCEL2},
+ {"KC_EXLM", KC_EXLM},
+ {"KC_AT", KC_AT},
+ {"KC_HASH", KC_HASH},
+ {"KC_DLR", KC_DLR},
+ {"KC_PERC", KC_PERC},
+ {"KC_CIRC", KC_CIRC},
+ {"KC_AMPR", KC_AMPR},
+ {"KC_ASTR", KC_ASTR},
+ {"KC_LPRN", KC_LPRN},
+ {"KC_RPRN", KC_RPRN},
+ {"KC_UNDS", KC_UNDS},
+ {"KC_PLUS", KC_PLUS},
+ {"KC_LCBR", KC_LCBR},
+ {"KC_RCBR", KC_RCBR},
+ {"KC_PIPE", KC_PIPE},
+ {"KC_COLN", KC_COLN},
+ {"KC_DQUO", KC_DQUO},
+ {"KC_TILD", KC_TILD},
+ {"KC_LT", KC_LT},
+ {"KC_GT", KC_GT},
+ {"KC_QUES", KC_QUES},
+//  {"FN_TT13", FN_TT13}, These are seemingly unused in QMK, but found in VIA for some reason
+//  {"FN_TT23", FN_TT23},
+//  {"SPC_FN1", SPC_FN1},
+//  {"SPC_FN2", SPC_FN2},
+//  {"SPC_FN3", SPC_FN3},
+ {"RESET", RESET},
+ {"DEBUG", DEBUG},
+ {"MAGIC_TOGGLE_NKRO", MAGIC_TOGGLE_NKRO},
+ {"KC_GESC", KC_GESC},
+ {"AU_ON", AU_ON},
+ {"AU_OFF", AU_OFF},
+ {"AU_TOG", AU_TOG},
+ {"CLICKY_TOGGLE", CLICKY_TOGGLE},
+ {"CLICKY_ENABLE", CLICKY_ENABLE},
+ {"CLICKY_DISABLE", CLICKY_DISABLE},
+ {"CLICKY_UP", CLICKY_UP},
+ {"CLICKY_DOWN", CLICKY_DOWN},
+ {"CLICKY_RESET", CLICKY_RESET},
+ {"MU_ON", MU_ON},
+ {"MU_OFF", MU_OFF},
+ {"MU_TOG", MU_TOG},
+ {"MU_MOD", MU_MOD},
+ {"BL_ON", BL_ON},
+ {"BL_OFF", BL_OFF},
+ {"BL_DEC", BL_DEC},
+ {"BL_INC", BL_INC},
+ {"BL_TOGG", BL_TOGG},
+ {"BL_STEP", BL_STEP},
+ {"BL_BRTG", BL_BRTG},
+ {"RGB_TOG", RGB_TOG},
+ {"RGB_MOD", RGB_MOD},
+ {"RGB_RMOD", RGB_RMOD},
+ {"RGB_HUI", RGB_HUI},
+ {"RGB_HUD", RGB_HUD},
+ {"RGB_SAI", RGB_SAI},
+ {"RGB_SAD", RGB_SAD},
+ {"RGB_VAI", RGB_VAI},
+ {"RGB_VAD", RGB_VAD},
+ {"RGB_SPI", RGB_SPI},
+ {"RGB_SPD", RGB_SPD},
+ {"RGB_M_P", RGB_M_P},
+ {"RGB_M_B", RGB_M_B},
+ {"RGB_M_R", RGB_M_R},
+ {"RGB_M_SW", RGB_M_SW},
+ {"RGB_M_SN", RGB_M_SN},
+ {"RGB_M_K", RGB_M_K},
+ {"RGB_M_X", RGB_M_X},
+ {"RGB_M_G", RGB_M_G},
+ {"KC_LSPO", KC_LSPO},
+ {"KC_RSPC", KC_RSPC},
+ {"KC_SFTENT", KC_SFTENT},
+ {"KC_LCPO", KC_LCPO},
+ {"KC_RCPC", KC_RCPC},
+ {"KC_LAPO", KC_LAPO},
+ {"KC_RAPC", KC_RAPC},
+//  {"BR_INC", BR_INC}, These are seemingly unused in QMK, but found in VIA for some reason
+//  {"BR_DEC", BR_DEC},
+//  {"EF_INC", EF_INC},
+//  {"EF_DEC", EF_DEC},
+//  {"ES_INC", ES_INC},
+//  {"ES_DEC", ES_DEC},
+//  {"H1_INC", H1_INC},
+//  {"H1_DEC", H1_DEC},
+//  {"S1_INC", S1_INC},
+//  {"S1_DEC", S1_DEC},
+//  {"H2_INC", H2_INC},
+//  {"H2_DEC", H2_DEC},
+//  {"S2_INC", S2_INC},
+//  {"S2_DEC", S2_DEC},
```

Comment:
These are for Wilba's implementation of RGB Matrix, which is mostly exclusive to his boards.
```suggestion
```

Reply:
Removed these and the other set of unused keycodes